src/core/loader/loader.cpp

```diff
@@ -114,7 +116,8 @@ static std::unique_ptr<AppLoader> GetFileLoader(FileUtil::IOFile&& file, FileTyp
     case FileType::CXI:
     case FileType::CCI:
         return std::make_unique<AppLoader_NCCH>(std::move(file), filepath);
-
+    case FileType::CIA:
+      return std::make_unique<AppLoader_CIA>(std::move(file), filepath);
```

Comment:
Given that CIA is also an NCCH container, you should probably extend the constructor for `AppLoader_NCCH` with a `FileType` argument or create wrappers for CXI / CCI to make this uniform somehow

I'm sure this would be brought up in review too. Personally I couldn't care less :P